src/stories/Popover.stories.tsx

```
@@ -0,0 +1,90 @@
+      </div>
+    ),
+  ],
+};
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  render: (args) => {
+    const [_, updateArgs] = useArgs();
```

ESLint

[error] '_' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)